Source/Core/DolphinQt2/GameList/GameList.cpp

```diff
@@ -399,6 +401,24 @@ QString GameList::GetSelectedGame() const
   return QStringLiteral("");
 }
 
+QStringList GameList::GetAllGamePaths() const
+{
+  QStringList paths;
+
+  QAbstractItemView* view;
+  if (currentWidget() == m_table)
+    view = m_table;
+  else
+    view = m_list;
+
+  auto* model = view->model();
```

Comment:
The get-model-from-view logic should be removed and replaced with simply `m_model`.